tests/memoryview/memslice.pyx

```diff
@@ -1,5 +1,5 @@
 # mode: run
-
+# tag: abcd
```

Comment:
?

Reply:
Apologies, that was quite careless.  Removed (and rebased on top of current master).